platform/pulpcore/app/tasks/publisher.py

```diff
@@ -3,11 +3,20 @@
 from celery import shared_task
 
 from pulpcore.app import models
+from pulpcore.app import serializers
 from pulpcore.tasking.services import storage
 from pulpcore.tasking.tasks import UserFacingTask
 
 
 @shared_task(base=UserFacingTask)
+def update(publisher_id, data, *args, **kwargs):
```

Comment:
Should data be a required argument? Is it allowable for the user to call PUT without passing in data?